src/index.html

```diff
@@ -1,15 +1,167 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form>
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <div class="item">
+          <label for="lname">Surname:</label>
+          <input
+            type="text"
+            id="lname"
+            name="lname"
+            autocomplete="off"
+            required>
+        </div>
+        <div class="item">
+          <label for="fname">Name:</label>
+          <input
+            type="text"
+            id="fname"
+            name="fname"
+            autocomplete="off">
+        </div>
+        <div class="item">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            class="age"
+            id="age"
+            name="age"
+            value="12">
+        </div>
+        <div class="item">
+          <label for="birth">Full date of birth:</label>
+          <input
+            type="date"
+            id="birth"
+            name="birth">
```

Comment:
```suggestion
          <input
            type="date"
            id="birth"
            name="birth"
          >
```
move all closing braces to the new line like this